### DISCUSSION TITLE
A thought on weather

### DISCUSSION THREAD

**Discussion by czimm74** [aaronwmorris/indi-allsky#964] [Ideas]
This is just a rambling idea in my head.  But been thinking some weather information would be useful, mainly I am thinking cloud cover / clear sky forecast.

Thinking this because I have been looking through the [Pirate Weather API](http://pirateweather.net/en/latest/).  Mainly as a quick way to see how the nights viewing would be.  Could also be something to tie into the dew control.  Again, just ideas rolling around in my head.

**Comment by czimm74** [comment]
Adding a note to myself.. But here is a way to include onto a page: https://github.com/ktrue/PW-forecast

**Comment by gsgghotra** [comment]
You can also use [https://openweathermap.org/api](https://openweathermap.org/api) which does provide a lot of data about the weather. It's free and very easy to use.

**Reply by aaronwmorris** [reply]
Its not a bad idea.  I will look into this.
